### ISSUE TITLE
log files

### ISSUE THREAD

**Issue by TMTYD** [webmin/webmin#792]
Hi

When I go to system, system logs, I see the log files, if I try now the log file of apache / error.log, try to edit, they can not open, because it is not linked, the same is synonymous with the miniserv.error.

My webmin version is 1.872.0125.2038

![log](https://user-images.githubusercontent.com/29516076/35597284-f78d8aba-061d-11e8-9c2b-f3209155c3a0.png)
![log 2](https://user-images.githubusercontent.com/29516076/35597288-f9201f1e-061d-11e8-9a05-faac388236f3.png)

**Comment by jcameron** [comment]
It's expected that they aren't linked, as they aren't written to by Syslog. However, you can still use the link on the right to view the logs.

**Comment by TMTYD** [comment]
That is correct, that I can click on the right button, if I z.b. when apache2 / error.log click on the right button, it also appears, but if I then select in the dropdown menu on the webmin / miniserv.error, it will not load and anderst rum. That was not the case before the update, the two files were also linked to edit them, and there are some messages in the miniserv.error.
